website/assets/js/pages/query-detail.page.js

```diff
@@ -3,23 +3,45 @@ parasails.registerPage('query-detail', {
   //  ║║║║║ ║ ║╠═╣║    ╚═╗ ║ ╠═╣ ║ ║╣
   //  ╩╝╚╝╩ ╩ ╩╩ ╩╩═╝  ╚═╝ ╩ ╩ ╩ ╩ ╚═╝
   data: {
-    //…
+    contributors: [],
   },
 
   //  ╦  ╦╔═╗╔═╗╔═╗╦ ╦╔═╗╦  ╔═╗
   //  ║  ║╠╣ ║╣ ║  ╚╦╝║  ║  ║╣
   //  ╩═╝╩╚  ╚═╝╚═╝ ╩ ╚═╝╩═╝╚═╝
-  beforeMount: function() {
+  beforeMount: function () {
     //…
   },
-  mounted: async function() {
-    //…
+  mounted: async function () {
+    if (this.query && this.query.contributors) {
```

Comment:
fyi I'd get rid of this check (you should be able to rely on the data being complete, and if it's not, we can ensure it's always normalized and in the proper format in the build script)

That said, we discussed moving this fetch to the build script anyway so no worries on changing it-- just bringing it up in the interest of philosophical alignment